test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ViewComponentTests.cs

```diff
@@ -140,6 +140,24 @@ public void ViewComponent_View_WithModelParameter_SetsResultViewWithDefaultViewN
         }
 
         [Fact]
+        public void ViewComponent_View_WithNullModelParameter_SetsResultViewWithDefaultViewNameAndNullModel()
+        {
+            // Arrange
+            var viewComponent = new TestViewComponent();
+
+            // Act
+            var actualResult = viewComponent.View(model: null as object);
```

Comment:
Suspect `viewComponent.View<object>(model: null)` would be more common. And, is in `ControllerTests`, might be worth checking `viewComponent.View<object>("something", model: null)`.

Should test `viewComponent<MyModel>("something", model: null)` since view components support that.

Do we already have tests confirming `viewComponent.View()` and perhaps `viewComponent.View(viewName: "something")` maintain `ViewModel.Model`?